engine/src/main/resources/assets/shaders/lightGeometryPass_frag.glsl

```diff
@@ -20,7 +20,9 @@ uniform vec3 lightViewPos;
 
 uniform sampler2D texSceneOpaqueDepth;
 uniform sampler2D texSceneOpaqueNormals;
+#if defined (FEATURE_LIGHT_DIRECTIONAL)
```

Comment:
No, not *necessary*.

That being said, digging through the entire shaders to find out why a particular varaible was being reported as unused was a chore. Since I wanted to make the code clearer for everyone (and myself for later on when I dived into it), I went ahead with it.